Resources/Prototypes/Traits/physical.yml

```
@@ -594,6 +594,10 @@
       inverted: true
       jobs:
         - Prisoner # Bionics should be "Confiscated" from long term prisoners.
+    - !type:CharacterSpeciesRequirement
+        inverted: true
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 8

(indentation)